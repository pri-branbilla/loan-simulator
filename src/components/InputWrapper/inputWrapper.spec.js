import React from 'react'
import { InputWrapper } from '.'
import renderer from 'react-test-renderer'

it('renders correctly with input', () => {
  const tree = renderer.create(
    <InputWrapper
      inputId="inputId"
      label="Test"
    >
      <input type="text" required name="inputId" id="inputId" value="14400" />
    </InputWrapper>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

it('renders correctly with select', () => {
  const tree = renderer.create(
    <InputWrapper
      inputId="inputId"
      label="Test"
    >
      <select name="inputId" id="inputId" required>
        <option value="1">1</option>
        <option value="2">2</option>
      </select>
    </InputWrapper>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
