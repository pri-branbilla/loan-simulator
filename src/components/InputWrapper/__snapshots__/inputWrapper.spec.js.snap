// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with input 1`] = `
<div
  className="field"
>
  <label
    htmlFor="inputId"
  >
    Test
  </label>
  <input
    id="inputId"
    name="inputId"
    required={true}
    type="text"
    value="14400"
  />
</div>
`;

exports[`renders correctly with select 1`] = `
<div
  className="field"
>
  <label
    htmlFor="inputId"
  >
    Test
  </label>
  <select
    id="inputId"
    name="inputId"
    required={true}
  >
    <option
      value="1"
    >
      1
    </option>
    <option
      value="2"
    >
      2
    </option>
  </select>
</div>
`;
